yaki_mobile/lib/presentation/displaydata/status_page_utils.dart

```diff
@@ -5,7 +5,7 @@ import 'package:yaki/domain/entities/declaration_status.dart';
 ///  Enum text (values) match the status format to be send to the API.
 enum StatusEnum {
   remote('remote'),
-  onSite('on site'),
+  office('office'),
```

Comment:
We need to use onSite('on site'), and not office('office'), as the value is used by the backend and we do key - value conversion (with toCamelCase function)